core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -317,7 +320,8 @@ oppia.directive('conversationSkin', [
           $scope.isLearnAgainButton = function() {
             var interaction = ExplorationPlayerService.getInteraction(
               $scope.activeCard.stateName);
-            if (INTERACTION_SPECS[interaction.id].is_linear) {
+            if ((interaction === null) ||
```

Comment:
Shouldn't the check be for "is currently in concept card mode"?

Reply:
Yeah, done.